addons/marker_flags/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,17 @@
+class Extended_PreStart_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_SCRIPT(XEH_preStart));
+    };
+};
+
+class Extended_PreInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_SCRIPT(XEH_preInit));
+    };
+};
+
+class Extended_PostInit_EventHandlers {
+    class ADDON {
+        clientInit = QUOTE(call COMPILE_SCRIPT(XEH_postInitClient));
```

Comment:
Would prefer just `XEH_postInit` which then has `hasInterface` check in it. But, that's just my personal preference.